plugins/inputs/ravendb/README.md

```diff
@@ -157,11 +157,11 @@ cluster.
     - storage_indexes_used_data_file_in_mb
     - storage_total_allocated_storage_file_in_mb
     - storage_total_free_space_in_mb
-    - storage_io_read_operations (optional, requires RavenDB v5.4.105+)
-    - storage_io_write_operations (optional, requires RavenDB v5.4.105+)
-    - storage_read_throughput_in_kb (optional, requires RavenDB v5.4.105+)
-    - storage_write_throughput_in_kb (optional, requires RavenDB v5.4.105+)
-    - storage_queue_length (optional, requires RavenDB v5.4.105+)
+    - storage_io_read_operations (optional, Linux only)
+    - storage_io_write_operations (optional, Linux only)
+    - storage_read_throughput_in_kb (optional, Linux only)
+    - storage_write_throughput_in_kb (optional, Linux only)
+    - storage_queue_length (optional, Linux only)
```

Comment:
Does this mean that these metrics have always been available, regardless of version, and that they are only supported on Linux?

Reply:
They're available for all supported versions of RavenDB, that's why I removed version requirement. We currently only expose them on Linux, for other platforms default value is assigned for each of these metrics.